script/people/migrate_ai_an_to_attested_status.rb

```diff
@@ -0,0 +1,56 @@
+# frozen_string_literal: true
+
+# This script identifies and updates the 'American Indian Status' verification type
+# fields in People, moving their state from 'outstanding', 'review', 'pending', 'unverified',
+# and 'rejected' to 'attested'.
+#
+# The script operates in two modes:
+# 1. `report`: Generates a report of people with 'American Indian Status' verification type fields in the aforementioned states.
+# 2. `data_fix`: Updates the 'American Indian Status' verification types.
+#
+# @example Run in report mode to identify people with negative American Indian Status states
+#   MODE="report" bundle exec rails runner script/people/migrate_ai_an_to_attested_status.rb
+#
+# @example Run in data_fix mode to update the American Indian Status verification types to attested
+#   MODE="data_fix" bundle exec rails runner script/people/migrate_ai_an_to_attested_status.rb
+
+mode = ENV['MODE']
```

Comment:
There is a slight operational concern with using an `ENV` variable to set the mode.  Often times, devops (or devs) will run scripts/migrations as background jobs using the `nohup` command.  It is not possible to pass an env var within the `nohup` command, for example: 
```
nohup MODE="data_fix" bundle exec rails runner script/people/migrate_ai_an_to_attested_status.rb &
```

Using `ARGV[0]` to access the first argument passed in after the script does work, however.  For example, the script could be run like this:
```
nohup bundle exec rails runner script/people/migrate_ai_an_to_attested_status.rb data_fix &
```
It is definitely a trade off b/c the meaning is very clear when using an env var, but it's important to be able to run this migration as a background job if it takes more than a few minutes to run.